macOS CI build seems to be broken since a couple of days

See e.g. https://github.com/ddnet/ddnet/actions/runs/3373254688/jobs/5597624023:
```
ninja: error: 'package_dmg', needed by 'CMakeFiles/package_default', missing and no known rule to make it
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -98,6 +98,7 @@ jobs:
         brew install pkg-config sdl2 ffmpeg python3 ninja molten-vk vulkan-headers glslang spirv-tools
         brew upgrade freetype
         pip3 install dmgbuild
+        echo /Library/Frameworks/Python.framework/Versions/3.11/bin >> $GITHUB_PATH
         sudo rm -rf /Library/Developer/CommandLineTools
 
     - name: Build in debug mode

